vuepress/docs/next/docs/curate/bundle-presentation-config.md

```diff
@@ -1,10 +1,11 @@
-# How to customize bundle info shown in Entando App Builder
+# Customize Bundle Info Shown in App Builder
+
+An example of how a bundle is displayed in the Local Hub of an Entando Application is shown below:
```

Comment:
Entando Application >> App Builder

this was confusing because i thought there was somewhere else that showed a list of bundles, not in the App Builder or rephrase to "This is how the app builder displays bundles..."